src/js/common/schemaform/FormSaved.jsx

```diff
@@ -31,22 +46,32 @@ class FormSaved extends React.Component {
     }
   }
 
+  handleVerify() {
+    handleVerify(this.props.user.login.verifyUrl);
+  }
+
   render() {
     const { profile } = this.props.user;
     const lastSavedDate = this.props.lastSavedDate;
     const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(this.props.formId));
+    const notVerified = profile.accountType !== 3;
```

Comment:
I think a comment or a constant for this would be helpful. It's not clear what 3 means without context.